packages/common/src/utils/environment-utils.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Environment related utils, that is information about the mode and system we're operating in.
+ */
+export class EnvironmentUtils {
+  /**
+   * Returns whether or not we're running using rust-ceramic
+   */
+  static useRustCeramic(): boolean {
+    return process.env.IPFS_FLAVOR !== 'go'
```

Comment:
now that we're making rust-ceramic the default for unit tests, we should make sure that just running `npm test` on a local checkout of js-ceramic still works without any extra env vars.  That probably means setting some expectation around the default location of ceramic-one, or having it in your PATH